MonMaraicher/Features/Map/FarmerService.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 final class FarmerService {
 
     // TODO: Write unit tests for this method
-    func loadFarmers() async throws -> Farmers {
+    func loadFarmers(latitude: Double, longitude: Double) async throws -> Farmers {
```

Comment:
The naming of that func deserve to be updated now with the new params youve added. Maybe something like 
`findFarmers(at location: Location)` or something else

Reply:
Where is the unit test? :)